### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

**Comment by yosuke-furukawa** [comment]
Nice! 
But `Template Strings` is not explained so much.
I would like to put new exercise like "TEMPLATE STRING LITERAL" .

**Comment by marocchino** [comment]
:+1:

File: exercises/class/solution/solution.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Character {
     return this.health_;
   }
   toString() {
-    return "x: " + this.x + " y: " + this.y + " health: " + this.health_;
+    return `x: ${this.x} y: ${this.y} health: ${this.health_}`;
```
Comment (claudiopro) [inline exercises/class/solution/solution.js]: Nice one!

File: exercises/class_extend/solution/solution.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ class Player extends Character {
     this.y += dy;
   }
   toString() {
-    return "name: " + this.name + " " + super.toString();
+    return `name: ${this.name} ${super.toString()}`;
```
Comment (claudiopro) [inline exercises/class_extend/solution/solution.js]: I guess this is much alike issue #15.

**Comment by YanMagale** [comment]
Nice job. Its simple to use a template strings add increament in this example is simple and important to ES6 learning.

**Comment by marocchino** [comment]
I just close this PR.
Because, This PR over 1 year and other PR cover this. 🐈

**Pull request closed without merging**